Examples/pixmaps.lisp

```diff
@@ -0,0 +1,172 @@
+         (repaint-sheet stream region))))))
+
+(defun paste-tester (object &rest args)
+  (declare (ignore args))
+  (alexandria:when-let ((sheet (event-sheet object)))
+    (and (panep sheet)
+         (member (pane-name sheet) '(app1 app2 pix1 pix2)))))
+
+(define-drag-and-drop-translator tr-select-region
+    (blank-area command blank-area pixmaps :feedback region-feedback
+                                           :tester paste-tester)
+    (object destination-object)
+  (let ((x1 (pointer-event-x object))
+        (y1 (pointer-event-y object))
+        (x2 (pointer-event-x destination-object))
+        (y2 (pointer-event-y destination-object)))
+    (when (> x1 x2) (rotatef x1 x2))
+    (when (> y1 y2) (rotatef y1 y2))
+    (region-feedback nil nil (event-sheet object) x1 y1 x2 y2 :highlight)
+    (let* ((frame *application-frame*)
+           (app1 (find-pane-named frame 'app1))
+           (app2 (find-pane-named frame 'app2))
+           (pix1 (find-pane-named frame 'pix1))
+           (pix2 (find-pane-named frame 'pix2))
+           (dest (with-input-context (`(or (blank-area ,app1)
+                                           (blank-area ,app2)
+                                           (blank-area ,pix1)
+                                           (blank-area ,pix2)) :override t)
+                     (object)
+                     (loop (read-gesture))
+                   (blank-area object))))
+      (region-feedback nil nil (event-sheet object) x1 y1 x2 y2 :unhighlight)
+      `(com-copy ,(pane-name (event-sheet object))
+                 ,(pane-name (event-sheet dest))
+                 ,x1 ,y1 ,(- x2 x1) ,(- y2 y1)
+                 ,(pointer-event-x dest)
+                 ,(pointer-event-y dest)))))
+
+(define-presentation-action action-repaint-sheet
+    (blank-area command pixmaps :gesture :describe)
+    (object)
+  (let ((sheet (event-sheet object)))
+    (when (typep sheet 'pixmap-sheet)
+      (reset-pixmap sheet))
+    (repaint-sheet sheet +everywhere+)))
+
+(define-pixmaps-command (com-copy :name "Copy")
+    ((arg1 '(member app1 app2 pix1 pix2))
+     (arg2 '(member app1 app2 pix1 pix2))
+     (from-x 'integer)
+     (from-y 'integer)
+     (width 'integer)
+     (height 'integer)
+     (to-x 'integer)
+     (to-y 'integer))
+  (let* ((frame *application-frame*)
+         (pane1 (find-pane-named frame arg1))
+         (pane2 (find-pane-named frame arg2))
+         (medium1 (if (typep pane1 'pixmap-sheet)
+                      (sheet-pixmap pane1)
+                      (sheet-medium pane1)))
+         (medium2 (if (typep pane2 'pixmap-sheet)
+                      (sheet-pixmap pane2)
+                      (sheet-medium pane2))))
+    (medium-copy-area medium1 from-x from-y width height medium2 to-x to-y)
+    (map () (lambda (p)
+              (repaint-sheet p +everywhere+))
+         (remove-duplicates (list pane1 pane2)))))
+
+(defun pixmaps (&optional (new-process nil))
```

Comment:
`new-process` is usually a keyword parameter. Maybe do that here as well for consistency?

Reply:
fixed